website/static/js/contribAdder.js

```diff
@@ -60,68 +79,96 @@ var AddContributorViewModel = oop.extend(Paginator, {
         self.results = ko.observableArray([]);
         self.contributors = ko.observableArray([]);
         self.selection = ko.observableArray();
+
+        self.contributorIDsToAdd = ko.pureComputed(function () {
+            return self.selection().map(function (user) {
+                return user.id;
+            });
+        });
+
         self.notification = ko.observable('');
         self.inviteError = ko.observable('');
         self.doneSearching = ko.observable(false);
         self.totalPages = ko.observable(0);
-        self.nodes = ko.observableArray([]);
         self.nodesToChange = ko.observableArray();
 
-        self.foundResults = ko.pureComputed(function() {
+        self.foundResults = ko.pureComputed(function () {
```

Comment:
Minor: no need to add these extra spaces; they only add noise to the diffs. That said, don't feel you need to remove them at this point.